core/src/test/java/org/hibernate/ogm/backendtck/associations/manytoone/ManyToOneTest.java

```diff
@@ -42,8 +42,8 @@ public void testUnidirectionalManyToOne() throws Exception {
 		session.persist( emmanuel );
 		session.persist( jerome );
 		session.flush();
-		assertThat( getNumberOfEntities( sessions ) ).isEqualTo( 3 );
-		assertThat( getNumberOfAssociations( sessions ) ).isEqualTo( expectedAssociations() );
+		assertThat( getNumberOfEntities( session ) ).isEqualTo( 3 );
+		assertThat( getNumberOfAssociations( session ) ).isEqualTo( expectedAssociations() );
 		transaction.commit();
 		assertThat( getNumberOfEntities( sessions ) ).isEqualTo( 3 );
```

Comment:
> Apart from giving me less things to review, of course, which is a very valid reason ;)
 
That's the main reason, I prefer to keep the amount of changes to a minimum. For this cases in particular, we might have some tests that don't have a session and in general there is nothing wrong about using the factory. This change is about giving an additional option for those scenario where you need to make some assertions inside a specific transaction.